packages/common/src/core/schemas/internal/metrics/ProjectUiSchemaMetrics.ts

```diff
@@ -60,12 +63,65 @@ export const buildUiSchema = (
               },
             },
           },
+          {
+            scope: "/properties/_metric/properties/valueType",
+            type: "Control",
+            labelKey: `${i18nScope}.fields.metrics.valueType.label`,
+            rule: SHOW_FOR_STATIC_RULE_ENTITY,
+            options: {
+              control: "hub-field-input-tile-select",
+              layout: "horizontal",
+              helperText: {
+                labelKey: `${i18nScope}.fields.metrics.valueType.helperText`,
+                placement: "bottom",
+              },
+              enum: {
+                i18nScope: `${i18nScope}.fields.metrics.valueType.enum`,
+              },
+            },
+          },
           {
             labelKey: `${i18nScope}.fields.metrics.value.label`,
             scope: "/properties/_metric/properties/value",
             type: "Control",
-            rule: SHOW_FOR_STATIC_RULE_ENTITY,
+            rule: SHOW_FOR_STATIC_AND_STRING_RULE_ENTITY,
```

Comment:
yeah, I don't have any problem/see any other way around overloading this field with 3 different `uiSchema` definitions based on the value of `valueType`. I think this is fine.